product_docs/docs/biganimal/release/free_trial/index.mdx

```diff
@@ -0,0 +1,91 @@
+- **One active cluster, unlimited inactive ones** 
+  - You can try different configurations by deleting the active cluster and creating a new one.
+  - Restore deleted clusters at any time within the trial period.
+  - Choose from either AWS or Azure cloud hosting, with two regions available for each (out of [all supported regions](/biganimal/latest/overview/03a_region_support))
+  
+    | Cluster Details | On Azure                                      | On AWS                                                |
+    | --------------- | --------------------------------------------- | ----------------------------------------------------- |
+    | Region          | Either US East 2 or West Europe (Netherlands) | Either US-East 1 (N. Virginia) or EU-West-1 (Ireland) |
+    | Instance Type   | Esv3 series 2 vCPU with 16 GB RAM             | r5.large                                              |
+    | Storage         | Premium SSD disks P1 4Gi                      | AWS GP3 Volume, disk size 4                           |
+    |                 |                                               |                                                       |
+- **Edit database configuration settings.**
+- **Connect to your cluster** via a public IP from psql, pgAdmin, or your favorite PostgreSQL-friendly application.
+- **Access to EDB enterprise-level support.**
+- **Use BigAnimal services for 14 days.** The countdown begins when you first provision a cluster. 
+
+(Numerous other instance types, storage types, regions, customizations and support options are available with a paid account.)
+
+Get started with your trial by signing up for a free EDB account.
+
+## Create an account
+
+If you haven't done so already, you'll need to [create your EDB account](detail/create_an_account). 
+
+Registering an account on enterprisedb.com gets you access to the BigAnimal trial, as well as EDB's training and software downloads. 
+Accounts are free, and registration takes less than a minute.
+
+## Create a cluster
+
+The Free Trial allows anybody to create a database cluster hosted on BigAnimal to try for 14 days. While the trial only allows one cluster to be active at a time, you are free to delete your cluster and create a new one. It's even possible to restore any cluster you've previously deleted.
+
+BigAnimal offers both standard PostgreSQL and EnterpriseDB PostgreSQL Advanced Server (EPAS) clusters. You can try out either using the latest PostgreSQL version (14) or one of the previous supported versions (11-13). Other configuration options are limited for the free trail.
+
+There are several equivalent ways to create a cluster...
+
+- [...Using the portal](detail/create_a_cluster/create_cluster_portal)
```

Comment:
the ellipses starting bulleted list items are a little distracting. Maybe just have a sentence? 

You can create a cluster using either the portal or the CLI.

Reply:
I implemented Dee Dee's suggestion